src/components/TeamFounder/FounderSection.styles.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components/macro";
+
+interface TeamStyledProps {
+    mobileView: boolean;
+}
+export const Paragraph = styled.p`
+  color: white;
```

Comment:
You can use one of the text classes in `App.css` to style this paragraph, such as `regularText--white`

Reply:
I recommend different styles for the description vs. name and affiliation

Reply:
@IsaiahA21 please see my comment above ^

I’d like to use the same styling classes for name and affiliate as the other profiles